dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/lifecycle/ProcessLifecycleMonitor.kt

```diff
@@ -13,7 +13,7 @@ import androidx.annotation.MainThread
 import java.util.concurrent.atomic.AtomicBoolean
 import java.util.concurrent.atomic.AtomicInteger
 
-internal class ProcessLifecycleMonitor(val callback: Callback) :
+internal class ProcessLifecycleMonitor(internal val callback: Callback) :
```

Comment:
in theory this change is not needed, because the class itself is already internal.